setup.cfg

```diff
@@ -27,6 +27,7 @@ py_modules = flake8_import_order
 install_requires =
     pycodestyle
     setuptools
+python_requires > = 3.9
```

Comment:
This syntax is incorrect for a setuptools config file. As far as I can tell this line is essentially ignored and the latest flake8-import-order version is still the one chosen on Python <=3.8.

The correct syntax would be
```
python_requires = >=3.9
```